shared/base/view.js

```diff
@@ -135,6 +135,10 @@ module.exports = BaseView = Backbone.View.extend({
    * Any options not to create data-attributes for.
    */
   nonAttributeOptions: ['id', 'className', 'tagName'],
+  /**
+   * Any options to serialize as JSON (instead of omitting).
+   */
+  jsonAttributeOptions: ['fetch_params'],
```

Comment:
with html it's unsafe if you're setting fetch params on the server -> client because it will end up not encoding it correctly and say `Object object` instead, you should be using `JSON.stringify(params)` then setting the result to be `fetch_params`